### DISCUSSION TITLE
Ability to add multiple spotify accounts

### DISCUSSION THREAD

**Discussion by ronluna** [music-assistant/support#2121] [Feature requests and ideas]
Coming from slimdevices (formerly squeezebox and now Lyrion) , it was pretty handy to be able to add multple spotify accounts so each user can access their favorites and playlist. would it possible to add this feature?

maybe a duplicate #435 ?

**Comment by OzGav** [comment]
https://github.com/orgs/music-assistant/discussions/435
https://github.com/orgs/music-assistant/discussions/456

https://music-assistant.io/music-providers/spotify/
